terraform/environments/electronic-monitoring-data/s3_main.tf

```diff
@@ -149,3 +149,83 @@ module "athena-s3-bucket" {
 
   tags = local.tags
 }
+
+# ----------------------------------
+# Unzipped Data Store and log bucket
+# ----------------------------------
+
+module "unzipped-s3-data-store" {
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket?ref=cadab51"
+
+  bucket_prefix      = "unzipped-files-data-store-"
+  versioning_enabled = true
+
+  # to disable ACLs in preference of BucketOwnership controls as per https://aws.amazon.com/blogs/aws/heads-up-amazon-s3-security-changes-are-coming-in-april-of-2023/ set:
+  ownership_controls = "BucketOwnerEnforced"
+  acl                = "private"
+
+  # Refer to the below section "Replication" before enabling replication
+  replication_enabled = false
+  # Below variable and providers configuration is only relevant if 'replication_enabled' is set to true
+  # replication_region                       = "eu-west-2"
+  providers = {
+    # Here we use the default provider Region for replication. Destination buckets can be within the same Region as the
+    # source bucket. On the other hand, if you need to enable cross-region replication, please contact the Modernisation
+    # Platform team to add a new provider for the additional Region.
+    # Leave this provider block in even if you are not using replication
+    aws.bucket-replication = aws
+  }
+
+  lifecycle_rule = [
+    {
+      id      = "main"
+      enabled = "Enabled"
+      prefix  = ""
+
+      tags = {
+        rule      = "log"
+        autoclean = "true"
+      }
+
+      transition = [
+        {
+          days          = 365
+          storage_class = "STANDARD_IA"
+          }, {
+          days          = 730
+          storage_class = "GLACIER"
+        }
+      ]
+
+      expiration = {
+        days = 1000
+      }
+
+      noncurrent_version_transition = [
+        {
+          days          = 30
+          storage_class = "STANDARD_IA"
+          }, {
+          days          = 90
+          storage_class = "GLACIER"
+        }
+      ]
+
+      noncurrent_version_expiration = {
+        days = 365
+      }
+    }
+  ]
+
+  tags = local.tags
+}
+
+
+
+module "unzipped_store_log_bucket" {
```

Comment:
should probably get this module using the MP module instead also

Reply:
good point